### PR TITLE
Update free-podcasts-screencasts-en.md

### PR DESCRIPTION
**Pull request by Dkeys4**
A podcast covering the JavaScript language and more broadly web and frontend technologies, with guests from the open-source community. It's hosted by OpenReplay, an open-source session replay tool for developers. The podcast is available on the Apple platform, Spotify, and Amazon Music. The Podcast is totally free and is for programmers of all levels and experience.

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: casts/free-podcasts-screencasts-en.md [hunk 1]
```diff
@@ -194,6 +194,7 @@
 * [Syntax](https://syntax.fm) - Wes Bos, Scott Tolinski (podcast)
 * [The JavaScript Show](http://javascriptshow.com) - Peter Cooper, Jason Seifer (podcast)
 * [The Vanilla JS Podcast](http://javascriptshow.com) - Chris Ferdinandi. (podcast)
+* [20 Min JS Podcast](https://20minjs.com/) - Agustinus Theodorus, Chris Bongers, Mark Volkmann, and others (podcast)
```
Comment (rng70) [inline casts/free-podcasts-screencasts-en.md:197]: move this line to line 177

**Review by rng70 — CHANGES_REQUESTED** [review]

File: casts/free-podcasts-screencasts-en.md (hunk 1 above)
Comment (rng70) [reply casts/free-podcasts-screencasts-en.md:197]: linter issue will be solved

File: courses/free-courses-ne.md [hunk 2]
```diff
@@ -34,7 +35,9 @@
 * [SQL Full Course In Nepali](https://www.youtube.com/watch?v=Lt52pYaoSR8&list=PL2OJkQtHPRicxyldFGNJRRG4WwNe0Kjqe&index=2) - Technology Channel
 
 
-#### C
+#### Svelte
+
+* [Svelte - Crash Course](https://www.udemy.com/cart/subscribe/course/5049082) - Demian Kostelny, Udemy
```
Comment (rng70) [inline courses/free-courses-ne.md:41]: you removed `#### C` which is required here. also always give two newline(linebreak) between to section.

**Review by rng70 — CHANGES_REQUESTED** [review]

File: casts/free-podcasts-screencasts-en.md (hunk 1 above)
Comment (rng70) [reply casts/free-podcasts-screencasts-en.md:197]: you should change this to pass linter check

**Comment by rng70** [comment]
@Dkeys4 the problem is in file [casts/free-podcasts-screencasts-en.md](https://github.com/EbookFoundation/free-programming-books/pull/10635/files/2ce0c7d3f502b0438dd859f278e30d44aa16872f#diff-89a3d7bd4411eb8f157242bae989fc4a074bc98ca67fecb3bb941baafd879cae) line 197. move that line to line 177

**Comment by rng70** [comment]
linter issue still persists. let me check on the issue and review your changes.

**Comment by rng70** [comment]
@Dkeys4 the problem is also in file `books/free-programming-books-bn.md` in your repository. please make sure that your repository is up-to-date with the upstream repository. you can sync your repository with the upstream from your GitHub UI and also from terminal. If you are not interested to sync with upstream then make the following chnages in `books/free-programming-books-bn.md`

* move line 4 to line 6
* swap line 23 and 24
* move line 60 to line 62 (means move line 61 to line 60, line 62 to  line 61 and move line 60 to line 62)

and then commit. hope it will resolve the linter issue.

**note: maybe this problem occurs due to wrong sync from the upstream(not sure what happened from your end) because that file does not appears in changed files lists so it was quite unexpected**

File: casts/free-podcasts-screencasts-en.md [hunk 3]
```diff
@@ -174,6 +174,7 @@
 
 ### JavaScript
 
+* [20 Min JS Podcast](https://20minjs.com/) - Agustinus Theodorus, Chris Bongers, Mark Volkmann, and others (podcast)
```
Comment (eshellman) [inline casts/free-podcasts-screencasts-en.md:177]:
```suggestion
```
added in #10980 from #10628

**Comment by eshellman** [comment]
this PR suffered from a broken build caused when we added bangla valiidation. Sorry for the trouble! the resource is added in #10980

**Pull request closed without merging**